analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/InitializationVectorShouldBeRandom.cs

```diff
@@ -183,9 +165,6 @@ argument.Expression switch
             private static bool HasNotInitializedIVConstraint(SymbolicValue value, ProgramState programState) =>
```

Comment:
If we want to comply with SA1204StaticElementsMustAppearBeforeInstanceElements than the static elements should go before all instance elements.

Reply:
We don't. We're currently dogfooding SA rules and I've raised this one to be [discussed/removed](https://sonarsource.slack.com/archives/C012KBFFYD6/p1607018101012200).